src/02-crm/companies.apib

```diff
@@ -116,6 +116,8 @@ Get details for a single company.
             + language: `nl` (string)
             + preferred_currency: `EUR` (string, nullable)
             + payment_term (PaymentTerm, nullable)
+            + invoicing_preferences (object)
```

Comment:
Q: it might be nullable?

Reply:
I think we should always return an object here to make sure people always get the same datatype.